src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/ButtonSamplePage.xaml

```diff
@@ -20,37 +20,37 @@
 						<!-- Button Contained -->
 						<smtx:XamlDisplay UniqueKey="Material_ButtonSamplePage_Contained"
 										  Style="{StaticResource XamlDisplayBelowStyle}">
-							<Button Content="CONTAINED"
+							<Button Content="Contained"
```

Comment:
we should probably expose an attached property to control this behavior
extensions:ControlExtensions.ContentCasing="None|Upper|Lower"

Reply:
ill make an issue for this in the future

Reply:
https://github.com/unoplatform/Uno.Themes/issues/662